src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -25,7 +30,92 @@ public static void Initialize()
             ImageAnalytics.Initialize();
         }
 
+        private static unsafe ISchema GetModelSchema(IExceptionContext ectx, TFGraph graph)
+        {
+            var res = new List<KeyValuePair<string, ColumnType>>();
+            var opTypeGetters = new List<MetadataUtils.MetadataGetter<DvText>>();
+            var inputOpsGetters = new List<MetadataUtils.MetadataGetter<VBuffer<DvText>>>();
+            var inputOpsLengths = new List<int>();
+            foreach (var oper in graph)
```

Comment:
>oper [](start = 25, length = 4)

nit: If you shorten operation to oper, you can just go even further to "op" option.